### PR TITLE
[video] Fix default select action

### PR DESCRIPTION
**Pull request by ksooo**
Fixes a problem reported at the forum: https://forum.kodi.tv/showthread.php?tid=371461

Fix confirmed working by one of the problem reporters.

Runtime-tested also by me on macOS and Android, latest Kodi master.

@enen92 can you please review the code changes?

### CODE REVIEW

File: xbmc/video/windows/GUIWindowVideoBase.h [hunk 1]
```diff
@@ -33,8 +33,7 @@ class CGUIWindowVideoBase : public CGUIMediaWindow, public IBackgroundLoaderObse
   bool OnMessage(CGUIMessage& message) override;
   bool OnAction(const CAction &action) override;
 
-  virtual void OnItemInfo(const CFileItem& fileItem, ADDON::ScraperPtr& scraper);
-
+  virtual bool OnItemInfo(const CFileItem& fileItem, ADDON::ScraperPtr& scraper);
```
Comment (enen92) [inline xbmc/video/windows/GUIWindowVideoBase.h:36]: Maybe add docs to this method? What the bool is expected to represent?

Comment (ksooo) [reply xbmc/video/windows/GUIWindowVideoBase.h:36]: Fair. Will add some docs.

Comment (ksooo) [reply xbmc/video/windows/GUIWindowVideoBase.h:44]: @enen92 done.

**Review by enen92 — APPROVED** [review]
Merge and see who screams ✔️

**Pull request merged**